scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -22,6 +22,7 @@
 import os
 import re
 import subprocess
+from decimal import Decimal
```

Comment:
Please fix the Codacy warnings here.

'decimal.Decimal' imported but unused (F401)